pkg/ko/publish/daemon_test.go

```diff
@@ -29,12 +29,19 @@ import (
 
 type MockImageLoader struct{}
 
+var Tags []string
```

Comment:
@jonjohnsonjr  this is the only thing that I felt that was a little hacky, do you have any suggestion here or is this ok?

Reply:
I think this approach is fine for now.